__tests__/formats/__snapshots__/scssMaps.test.snap.js

```diff
@@ -51,3 +51,53 @@ $tokens: (
 `;
 /* end snapshot scss/map-deep snapshot */
 
+snapshots["should respect formatting.prefix option for scss/map-flat"] = 
+`
+/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+$tokens: (
+  '$foo-size-font-small': 12rem,
+  '$foo-size-font-large': 18rem,
+  // comment
+  '$foo-color-base-red': #ff0000,
+  '$foo-color-white': #ffffff,
+  '$foo-asset-icon-book': url("data:image/png;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJmZWF0aGVyIGZlYXRoZXItYm9vayI+PHBhdGggZD0iTTQgMTkuNUEyLjUgMi41IDAgMCAxIDYuNSAxN0gyMCI+PC9wYXRoPjxwYXRoIGQ9Ik02LjUgMkgyMHYyMEg2LjVBMi41IDIuNSAwIDAgMSA0IDE5LjV2LTE1QTIuNSAyLjUgMCAwIDEgNi41IDJ6Ij48L3BhdGg+PC9zdmc+")
+);`;
+/* end snapshot should respect formatting.prefix option for scss/map-flat */
+
+snapshots["should respect formatting.prefix option for scss/map-deep"] = 
+`
+/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+$foo-size-font-small: 12rem !default;
+$foo-size-font-large: 18rem !default;
+$foo-color-base-red: #ff0000 !default; // comment
+$foo-color-white: #ffffff !default;
+$foo-asset-icon-book: url("data:image/png;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJmZWF0aGVyIGZlYXRoZXItYm9vayI+PHBhdGggZD0iTTQgMTkuNUEyLjUgMi41IDAgMCAxIDYuNSAxN0gyMCI+PC9wYXRoPjxwYXRoIGQ9Ik02LjUgMkgyMHYyMEg2LjVBMi41IDIuNSAwIDAgMSA0IDE5LjV2LTE1QTIuNSAyLjUgMCAwIDEgNi41IDJ6Ij48L3BhdGg+PC9zdmc+") !default;
+
+$tokens: (
+  'size': (
+    'font': (
+      'small': $foo-size-font-small,
```

Comment:
these were missing the `$foo-` prefix before this change. Meaning the SCSS result is broken (broken refs)